### DISCUSSION TITLE
[Feature] version doubt

### DISCUSSION THREAD

**Discussion by lifegit** [yetone/bob-plugin-openai-translator#113] [Q&A] [answered]
### 描述 Motivation

你好，请问下0.x版本和1.x版本区别是什么？

最新版本：
v0.4.5（Nov 22, 2023）
v1.1.4（Nov 7, 2023）

**Comment by liby** [comment, marked as answer]
v0.4.x 对应的是[ Bob 1.0.0 以下的版本（不包括 1.0.0，例如 0.10.x 等版本）发布在 GitHub/Gitee 平台，简称社区版](https://bobtranslate.com/faq/appstore-migration.html#%E5%A6%82%E4%BD%95%E8%BF%81%E7%A7%BB%E5%88%B0-mac-app-store-%E7%89%88%E6%9C%AC)，处在维护状态，不会有重大功能更新。


v1.x 和 v2.x 对应的是 Bob 1.0.0 以上的版本，会在 Bob 支持的范围内进行功能更新。
